cmd/harmony/main.go

```diff
@@ -531,6 +538,8 @@ func createGlobalConfig(hc harmonyConfig) (*nodeconfig.ConfigType, error) {
 		nodeConfig.WebHooks.Hooks = config
 	}
 
+	nodeConfig.NtpServer = hc.Sys.NtpServer
```

Comment:
Since ntp check is done only once, do you think it would be better to put local system check in function `runHarmonyNode` (cmd/harmony/main.go:125) instead of adding more complexity to already exploded module `nodeConfig`?

Reply:
ntpServer may still be used later in get current time, so we need it in a configuration variable. Any suggestion on where to put it?

Reply:
I think after initial check,  we can use `time.Now()` as local current time. So why do we need ntpServer when after initialization?

Reply:
I still want to use ntp time when view change started, and default to time.Now(). The local maybe off, but within our range during initialization.  It may get off gradually later, since view change will be rely on time, I may still try to get ntp time during view change process.

Reply:
We shouldn't blindly trust a centralized service to set the local node's time. There will be attack vector from that. Ethereum just bring up a warning after checking with ntp, but ntp is not blindly used to set local time I think. It's the validator's responsibility to set their local time correctly. We can add warning about time drift. But eventually, how to set the local time shouldn't be automatically controlled by a centralized service.